omnibus/files/private-chef-cookbooks/private-chef/templates/default/openssl.cnf.erb

```diff
@@ -0,0 +1,361 @@
+#
+# OpenSSL example configuration file.
```

Comment:
@jaym if it's not too much work, I'd think that would be nicer... but that's just an opinion, really...